public-ui/src/app/data-browser/views/genomic-view/components/genomic-chart.component.tsx

```diff
@@ -1,34 +1,93 @@
+import { baseOptions } from 'app/data-browser/charts/react-base-chart/base-chart.service';
 import { reactStyles } from 'app/utils';
+import { genomicsApi } from 'app/services/swagger-fetch-clients';
+import * as highCharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
 import * as React from 'react';
-
 const styles = reactStyles({
-    border: {
-        border: '1px solid',
-        margin: '1rem'
+    chartContainer: {
+        background: 'rgba(33,111,180,0.05)',
+        margin: '1em',
+        padding: '1em',
+        paddingTop: '.25em'
+    },
+    chartTitle: {
+        fontSize: '1em'
     }
 });
 
 // tslint:disable-next-line:no-empty-interface
 interface Props {
-
+    data: any;
+    title: string;
 }
 // tslint:disable-next-line:no-empty-interface
 interface State {
-
+    options: any;
+    setData: any;
 }
 
 
 
 export class GenomicChartComponent extends React.Component<Props, State> {
+    chartOptions: any = [];
+
+
     constructor(props: Props) {
         super(props);
+        this.state = {
+            options: null,
+            setData: props.data
+        }
+        console.log(this.props.data, 'from', this.props.title);
+    }
+    
+
+    dataToOptions() {
+        this.chartOptions = baseOptions;
```

Comment:
I think the data is getting passed back to the `baseOptions` constant, so all three get left with the last set of data. Seems to work if I make sure the reference is broken using `JSON.parse`/`JSON.stringify`:
```suggestion
        const chartOptions = JSON.parse(JSON.stringify(baseOptions));
```